client/settings/express-checkout/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import ExpressCheckoutsSettings from './express-checkout-settings';
+
+const container = document.getElementById(
+	'wc_stripe-express_checkouts_customizer-container'
+);
+
+if ( container ) {
+	ReactDOM.render( <ExpressCheckoutsSettings />, container );
```

Comment:
Great work with the `methodId` cleanup! Could you please create an issue in the WCPay repo to also implement this change so we can keep these things "in sync" or at least to have a reference of why the difference is there?

Reply:
Created here:
https://github.com/Automattic/woocommerce-payments/issues/2863